### DISCUSSION TITLE
Export attribute required parameter to indicate warnings

### DISCUSSION THREAD

**Discussion by lamossus** [godotengine/godot-proposals#11353] [Editor]
I suggest adding another bool parameter to Export attribute called required that would show warning in inspector if exported value is not set. It would make it easier to at first glance see whether component is properly configured without having to constantly write samey _GetConfigurationWarnings functions as I often find myself writing them when exporting other components

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/550.
